DemoApp/en.lproj/Localizable.strings

```diff
@@ -51,12 +51,22 @@
 "message.sending.attachment-uploading-failed" = "UPLOADING FAILED";
 
 "message.title.online" = "Online";
-"message.title.see-minutes-ago" = "Seen %@ ago";
 "message.title.offline" = "Offline";
 "message.title.group" = "%d members, %d online";
 
 "current-selection" = "%d of %d";
 
 "attachment.max-size-exceeded" = "Attachment size exceed the limit.";
-
+"dates.time-ago-seconds-plural" = "%d seconds ago";
```

Comment:
I think we should keep "Seen" word for these... it's weird to see just "x months ago" without the verb "Seen"